server/base/src/main/java/org/apache/accumulo/server/conf/ServerConfigurationFactory.java

```diff
@@ -140,4 +158,114 @@ public void connectionEvent() {
       // no-op. changes handled by prop store impl
     }
   }
+
+  private class ConfigRefreshRunner {
+    private static final long MIN_JITTER_DELAY = 1;
+
+    private static final long MAX_JITTER_DELAY = 23;
+    private final ScheduledFuture<?> refreshTaskFuture;
+
+    ConfigRefreshRunner() {
+
+      Runnable refreshTask = this::verifySnapshotVersions;
+      // staggering the initial delay prevents synchronization of Accumulo servers communicating
+      // with ZooKeeper for the sync process.
+      long randDelay = jitter(REFRESH_PERIOD_MINUTES / 4, REFRESH_PERIOD_MINUTES);
+
+      ScheduledThreadPoolExecutor executor = ThreadPools.getServerThreadPools()
+          .createScheduledExecutorService(4, "config-refresh", false);
```

Comment:
Only have one task to repeat, so we only need 1 thread in the pool.
```suggestion
          .createScheduledExecutorService(1, "config-refresh", false);
```

Reply:
Fixed in ac14691996